### DISCUSSION TITLE
Fonction

### DISCUSSION THREAD

**Discussion by Julien4135** [robske110/IDDataLogger#41] [Q&A] [answered]
Hello, can the greyed-out functions in the Car.js code work if I remove the / before the code?

![image](https://user-images.githubusercontent.com/93113346/148542859-d311991b-cc38-4d17-afd7-3614c7f3ffd5.png)

**Comment by robske110** [comment, marked as answer]
No, since it will only be triggered when the widget is refreshed, therefore it is not of much use.
